colorbleed/plugins/maya/publish/validate_render_single_camera.py

```diff
@@ -18,33 +15,19 @@ class ValidateRenderSingleCamera(pyblish.api.InstancePlugin):
     """
 
     order = colorbleed.api.ValidateContentsOrder
+    label = "Render Single Camera"
     hosts = ['maya']
     families = ['colorbleed.renderlayer',
                 "colorbleed.vrayscene"]
-    label = "Render Single Camera"
-    actions = [colorbleed.maya.action.SelectInvalidAction]
```

Comment:
Let's put this action back in so artists can automatically select which cameras are problematic. :)